src/plugins/orangehrmCorePlugin/Api/V2/RequestParams.php

```diff
@@ -228,12 +230,29 @@ public function getAttachmentOrNull(string $type, string $key, ?array $default =
     {
         $attachment = $this->$type->get($key, $default);
         if (isset($attachment['name']) && isset($attachment['type']) && isset($attachment['base64']) && isset($attachment['size'])) {
-            return new Base64Attachment(
+            $attachment = new Base64Attachment(
                 $attachment['name'],
                 $attachment['type'],
                 $attachment['base64'],
                 $attachment['size']
             );
+
+            // Check for SVG and sanitize
+            if ($attachment->getFileType() === 'image/svg+xml') {
```

Comment:
have a look on these mime types also `text/xml`, `application/svg+xml`, `image/svg`

Reply:
seems image/svg+xml is the main one. our API also only allows for that

https://www.w3.org/TR/SVG11/intro.html#MIMEType
https://datatracker.ietf.org/doc/html/rfc3023#section-8.19
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types